src/commcare_cloud/ansible/roles/supervisor/tasks/main.yml

```diff
@@ -1,17 +1,30 @@
 ---
 
+- name: remove supervisor 3.0b
+  apt: name=supervisor state=absent
+  become: yes
+
 - name: find supervisord path
   shell: 'which supervisord'
   register: which_supervisord
   changed_when: false
   failed_when: which_supervisord.rc != 0 and which_supervisord.rc != 1
   check_mode: no
 
-- name: install supervisor
-  apt: name=supervisor state=present
+- name: install supervisor 3.3.4
+  pip: name=supervisor state=present version=3.3.4
```

Comment:
@dannyroberts it looks like the staging machines are consistently missing meld3 which is a dependency of this library. I checked swiss (deployed today) and it does exist there. Do you recall any issues that you had when deploying this?

Reply:
FWIW the output of running this command is:

```
Requirement already satisfied (use --upgrade to upgrade): supervisor==3.3.4 in /usr/local/lib/python2.7/dist-packages
Downloading/unpacking meld3>=0.6.5 (from supervisor==3.3.4)
  Downloading meld3-1.0.2-py2.py3-none-any.whl
Installing collected packages: meld3
Successfully installed meld3
```

meaning the correct version of supervisor was installed, but without the meld3 dependency